integ-test/src/test/java/org/opensearch/sql/ppl/QueryStringIT.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.ppl;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_BEER;
+
+import java.io.IOException;
+import org.json.JSONObject;
+import org.junit.Test;
+
+public class QueryStringIT extends PPLIntegTestCase {
+
+  @Override
+  public void init() throws IOException {
+    loadIndex(Index.BEER);
+  }
+
+  @Test
```

Comment:
We should include a test for "all fields" 
```
query_string(["*"], ...
```

Reply:
Resolved under [7f6b006082c8de619b8a0ea1f0190dcbd5934532](https://github.com/Bit-Quill/opensearch-project-sql/pull/85/commits/7f6b006082c8de619b8a0ea1f0190dcbd5934532)